.vscode/settings.json

```diff
@@ -16,6 +16,13 @@
   "javascript.format.enable": false,
   "typescript.format.enable": false,
 
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": false,
+  "editor.codeActionsOnSave": {
+    "source.formatDocument": true,
```

Comment:
Whats the diff between editor.formatOnSave vs source.formatDocument?

Reply:
`formatOnSave` only applies the default formatter I believe vs using `codeActionsOnSave` which allows us to do multiple actions on save. In this case, `formatDocument` applies prettier fixes and then `fixAll` applies eslint fixes